tooling/cli/src/plugin.rs

```diff
@@ -25,17 +28,46 @@ pub struct Cli {
 
 #[derive(Subcommand)]
 enum Commands {
+  New(new::Options),
```

Comment:
I prefer `create` instead of `new` what do you think?

Reply:
`new` is the usually the term I see in bootstrapping CLIs when you have both `init` and `new`, for example, `cargo new` and `cargo init` (I honestly can't think of another example, but I am quite certain I saw it somewhere else)

Reply:
or we could have both